js/msp.js

```diff
@@ -406,11 +408,7 @@ var MSP = {
                     RC_tuning.RC_YAW_EXPO = parseFloat((data.getUint8(offset++) / 100).toFixed(2));
                     if (semver.gte(CONFIG.apiVersion, "1.20.0")) {
                         RC_tuning.rcYawRate = parseFloat((data.getUint8(offset++) / 100).toFixed(2));
-                    } else {
-                        RC_tuning.rcYawRate = 0;
```

Comment:
Not entirely true. This only applies to `if (semver.gte(CONFIG.flightControllerVersion, "2.9.0") && semver.lt(CONFIG.flightControllerVersion, "3.0.0"))`. We should reset this for versions < 2.9.0.